src/lib.rs

```diff
@@ -30,7 +31,8 @@ pub struct Configuration {
     total_steps: usize,
     users_per_step: usize,
     friendship_ratio: f32,
-    step_duration_in_secs: usize,
+    step_duration_in_secs: u64,
+    tick_duration_in_secs: u64,
```

Comment:
QQ: Can't these be `Duration`s instead? I think this was possible.

Reply:
will tackle this on next PR 👍